backend/src/main/java/ubb/courses/backend/models/Course.java

```diff
@@ -0,0 +1,55 @@
+package ubb.courses.backend.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Set;
+
+@Entity
+@Table(name = "courses")
+@EntityListeners(AuditingEntityListener.class)
+public class Course extends BaseEntity<Long> implements Serializable {
+    @NotBlank
+    private String title;
+
+    @Column(nullable = false, updatable = false)
+    @Temporal(TemporalType.TIMESTAMP)
+    @CreatedDate
+    private Date createdAt;
```

Comment:
Good job! Please add a last modified date as well. Also some description should be attributed to the course entity.